### ISSUE TITLE
Linked Buffers Fail to Update from Remote

### ISSUE THREAD

**Issue by xeno-by** [djspiewak/cccp#3]
I've built CCCP with a few amendments (https://github.com/xeno-by/cccp) and then did the following:
1) Ran the server (namely, "%PROJECTS%\cccp\server\dist\bin\cccp-server.bat" --configFile "%PROJECTS%\cccp\server\default.conf")
2) Ran the first instance of jEdit ("%JEDIT_HOME%\jedit.exe" -noserver)
3) Opened build.xml from scala/scala ("%PROJECTS%\Scala\build.xml")
4) Ran the second instance of jEdit ("%JEDIT_HOME%\jedit.exe" -noserver)
5) Opened build.xml from my fork of scala/scala ("%PROJECTS%\Kepler\build.xml")
6) In both instances of jEdit invoked "Plugins > CCCP > Link file..." and specified the very same file ID.
7) Started editing the first file, but the second instance of jEdit didn't get updated

Does jEdit plugin need any configuration after being copied to "%JEDIT_HOME%/jars"? How do I make sure that the server is running correctly? Any clues on debugging the CCCP protocol? And, finally, the Ultimate Question of Life, the Universe, and Everything: "omg it does not work! fix that for me!!".

My environment is as follows:

```
Microsoft Windows [Version 6.1.7601]
java version "1.6.0_25"
Java(TM) SE Runtime Environment (build 1.6.0_25-b06)
Java HotSpot(TM) 64-Bit Server VM (build 20.0-b11, mixed mode)
Scala code runner version 2.9.1.final -- Copyright 2002-2011, LAMP/EPFL
```

**Comment by djspiewak** [comment]
I think the correct answer here is "OMG, it does not work".  :-)  Working on this today.

**Comment by djspiewak** [comment]
I can confirm this issue.  There's something funky going on in the agent w.r.t. server ops.  I'm tracking it down now.

**Comment by djspiewak** [comment]
Oddly, this issue is somewhat intermittent.  I added some logging to the agent and it went away.  :-)  I did notice that there was an invalid assertion in the agent, which I removed.  I don't think that was it though.

One thing that's probably worth inspecting is the agent logs.  The jEdit plugin is actually hard-coded to dump these into /tmp.  I'm not sure exactly what that means for Windows, but that's where they would go!  It's entirely possible that the agent startup process actually failed at that point and you never got to link your buffers.  We might try addressing this hard coding and then trying again.
